### PR TITLE
Update to v3.3.

### PR DESCRIPTION
**Pull request by mkosola**

### CODE REVIEW

**Review by abranson — APPROVED** [review]

**Comment by saukko** [comment]
Please move this also to sailfishos org in github. We should not use this organization anymore.

**Comment by mkosola** [comment]
@saukko Ok, will move after merge

File: rpm/systemtap.spec [hunk 1]
```diff
@@ -228,7 +228,7 @@ done
 # We build it in place and then move it away so it doesn't get installed
 # twice. rpm can specify itself where the (versioned) docs go with the
 # %doc directive.
-mv $RPM_BUILD_ROOT%{_datadir}/doc/systemtap/examples examples
+mv $RPM_BUILD_ROOT/usr/share/systemtap/examples examples
```
Comment (rainemak) [inline rpm/systemtap.spec:231]: Btw, why did you drop datadir as that should / must point of /usr/share ?

Comment (mkosola) [reply rpm/systemtap.spec:231]: yeah, should have dropped only /doc

**Pull request merged**